tasks/jasmine.js

```diff
@@ -194,7 +195,7 @@ module.exports = function(grunt) {
 
     phantomjs.on('console', function(msg) {
       thisRun.cleanConsole = false;
-      if (options.display === 'full') {
+      if (options.showLogging) {
```

Comment:
> I thought you were against the use of `options.display` to determine whether logs should show

I am. I personally don't care if backwards compatibility is broken, because I don't think the original behavior was desirable.

I am am offering to viable solutions, one is progressive and one is conservative.

 1. **Progressive**: Ignore `display` when logging and default to `showLogging` to true.
 2. **Conservative**: Default `showLogging` to undefined and have that state preserve backwards compatibility.

I have a feeling that the maintainers will lean towards the conservative option. They may lean towards a third option; radical conservatism: don't increase the options complexity at all.

It's not up to me, I'm just offering logical alternatives to facilitate progress.